src/api/composition/CompositionAPI.js

```diff
@@ -20,56 +20,59 @@
  * at runtime from the About dialog for additional information.
  *****************************************************************************/
 
-define([
-    'lodash',
-    'EventEmitter',
-    './DefaultCompositionProvider',
-    './CompositionCollection'
-], function (
-    _,
-    EventEmitter,
-    DefaultCompositionProvider,
-    CompositionCollection
-) {
+import CompositionProvider from './CompositionProvider';
+import CompositionCollection from './CompositionCollection';
+
+/**
+ * @typedef {import('../objects/ObjectAPI').DomainObject} DomainObject
+ */
+
+/**
+ * @typedef {import('../../../openmct').OpenMCT} OpenMCT
+ */
+
+/**
+ * An interface for interacting with the composition of domain objects.
+ * The composition of a domain object is the list of other domain objects
+ * it "contains" (for instance, that should be displayed beneath it
+ * in the tree.)
+ * @constructor
+ */
+export default class CompositionAPI {
     /**
-     * An interface for interacting with the composition of domain objects.
-     * The composition of a domain object is the list of other domain objects
-     * it "contains" (for instance, that should be displayed beneath it
-     * in the tree.)
-     *
-     * @interface CompositionAPI
-     * @returns {module:openmct.CompositionCollection}
-     * @memberof module:openmct
+     * @param {OpenMCT} publicAPI
      */
-    function CompositionAPI(publicAPI) {
+    constructor(publicAPI) {
+        /** @type {CompositionProvider[]} */
         this.registry = [];
+        /** @type {CompositionPolicy[]} */
         this.policies = [];
-        this.addProvider(new DefaultCompositionProvider(publicAPI, this));
+        this.addProvider(new CompositionProvider(publicAPI, this));
+        /** @type {OpenMCT} */
         this.publicAPI = publicAPI;
     }
-
     /**
      * Add a composition provider.
      *
      * Plugins can add new composition providers to change the loading
      * behavior for certain domain objects.
      *
      * @method addProvider
-     * @param {module:openmct.CompositionProvider} provider the provider to add
+     * @param {CompositionProvider} provider the provider to add
      * @memberof module:openmct.CompositionAPI#
```

Comment:
I went back and forth on this but decided to keep it in case JSDoc + TypeScript decides to support it in the future. [There's actually an open RFE ](https://github.com/Microsoft/TypeScript/issues/7237)for adding `@memberof` support